.eleventy.js

```diff
@@ -76,6 +78,20 @@ module.exports = function (eleventyConfig) {
     return lastDate.toISOString();
   });
 
+eleventyConfig.addShortcode("formatAuthors", authors => {
+  const authorArray = Array.isArray(authors)
+    ? authors
+    : [...authors.split(", ")];
```

Comment:
Maybe split on `,` instead of `, ` and then trim each item. This way, even if we forget to add a space after `,` in the md file, the split will still work. Seems a bit more robust.

Reply:
updated!